addons/sales_team/static/src/js/sales_team.js

```diff
@@ -13,7 +13,7 @@ KanbanRecord.include({
      * @private
      */
     _openRecord: function () {
-        if (this.modelName === 'crm.team') {
+        if (this.modelName === 'crm.team' && $(".oe_kanban_crm_salesteams_list").length != 0) {
```

Comment:
I think that .oe_kanban_crm_salesteams_list is not used anymore.
We should delete this file.